style.css

```diff
@@ -423,3 +426,65 @@ button {
   font-size: 32px;
   line-height: 44px;
 }
+
+#Contact-us {
+  margin-top: 120px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.interest {
+  text-align: center;
+  margin-bottom: 73px;
+  font-weight: 700;
+  font-size: 32px;
+  font-family: 'Inter', sans-serif;
+  color: #172b4d;
+  font-style: normal;
+  line-height: 44px;
+  width: 327px;
+  height: 216px;
+}
+
+.text1 {
+  border: none;
+  border-bottom: 1px solid black;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  margin-left: 40px;
```

Comment:
- [ ] You did a good job however, it is required to use flexbox to layout the form elements

Reply:
Thank you @algerina. I've made all changes as requested